File: filmy-app/app/src/main/java/rk/entertainment/filmy/models/moviesDetails/VideosData.java
package rk.entertainment.filmy.models.moviesDetails;

import com.google.gson.annotations.SerializedName;

public class VideosData {

    @SerializedName("id")
    private String id;

    @SerializedName("iso_639_1")
    private String iso6391;

    @SerializedName("iso_3166_1")
    private String iso31661;

    @SerializedName("key")
    private String key;

    @SerializedName("name")
    private String name;

    @SerializedName("site")
    private String site;

    @SerializedName("size")
    private Integer size;

    @SerializedName("type")
    private String type;


    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getIso6391() {
        return iso6391;
    }

    public void setIso6391(String iso6391) {
        this.iso6391 = iso6391;
    }

    public String getIso31661() {
        return iso31661;
    }

    public void setIso31661(String iso31661) {
        this.iso31661 = iso31661;
    }

    public String getKey() {
        return key;
    }

    public void setKey(String key) {
        this.key = key;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getSite() {
        return site;
    }

    public void setSite(String site) {
        this.site = site;
    }

    public Integer getSize() {
        return size;
    }

    public void setSize(Integer size) {
        this.size = size;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }
}

File: filmy-app/app/src/main/java/rk/entertainment/filmy/data/models/moviesDetails/VideosData.kt
package rk.entertainment.filmy.data.models.moviesDetails

import com.google.gson.annotations.SerializedName

data class VideosData(
        @SerializedName("id")
        var id: String? = null,

        @SerializedName("iso_639_1")
        var iso6391: String? = null,

        @SerializedName("iso_3166_1")
        var iso31661: String? = null,

        @SerializedName("key")
        var key: String? = null,

        @SerializedName("name")
        var name: String? = null,

        @SerializedName("site")
        var site: String? = null,

        @SerializedName("size")
        var size: Int? = null,

        @SerializedName("type")
        var type: String? = null
)